modin/engines/ray/utils.py

```diff
@@ -103,7 +103,7 @@ def initialize_ray(
             # We only start ray in a cluster setting for the head node.
             ray.init(
                 address=redis_address or "auto",
-                include_webui=False,
+                include_dashboard=False,
```

Comment:
change from `include_webui` to `include_dashboard` due to this pr in ray community  https://github.com/ray-project/ray/pull/8999